packages/synapse-react-client/src/components/ChallengeSubmission/ChallengeSubmission.tsx

```diff
@@ -120,70 +117,71 @@ export function ChallengeSubmission({
     useErrorBoundary: true,
   })
 
-  useGetEntityAlias(newProject?.alias ?? EMPTY_ID, {
-    enabled: newProject !== undefined && !!challenge && !!submissionTeam,
-    onSettled: (
-      data: EntityId | null | undefined,
-      error: SynapseClientError | null,
-    ) => {
-      if (data) {
-        setProjectAliasFound(true)
-        setChallengeProjectId(data.id)
-      }
-      if (error) {
-        setProjectAliasFound(false)
-      }
```

Comment:
I might've missed it, but I didn't see the error handled in the updated code. Is the error handling still needed?